data/talkactions/scripts/start_cast.lua

```diff
@@ -0,0 +1,10 @@
+function onSay(cid, words, param)
+	local player = Player(cid)
+	if player:startLiveCast(param) then
+		player:sendTextMessage(MESSAGE_INFO_DESCR, "You have started casting your gameplay.")
+		return false
```

Comment:
You can move "return false" to the end of the function and remove the two current instances of it, since the function returns false either way. Also add a new line at the end of the file. This applies to stop_cast.lua too.